package.json

```diff
@@ -21,8 +21,9 @@
     "docs:deploy:manual": "npm --prefix docs run deploy:manual",
     "docs:dev": "npm --prefix docs run dev",
     "github-release": "github-release-from-changelog",
-    "lint": "yarn lint:js . && yarn lint:md .",
+    "lint": "yarn lint:js . && yarn lint:ts && yarn lint:md .",
     "lint:js": "cross-env NODE_ENV=production eslint --cache --cache-location=.cache/eslint --ext .js,.jsx,.json",
+    "lint:ts": "tslint -p . -t stylish -e '{**/node_modules/**,**/dist/**,lib/cli/test/**,**/storybook-static/**}' **/*.ts",
```

Comment:
do they have something like `.tslintignore`?

Reply:
Right, I'll check it!